FAILURE! - in io.druid.query.lookup.LookupSnapshotTakerTest

When I run 'mvn test' on Building druid-processing 0.9.1-SNAPSHOT one test fails

Failed tests:
  LookupSnapshotTakerTest.testIOExceptionDuringLookupPersist Expected exception: com.metamx.common.ISE

Tests run: 8868, Failures: 1, Errors: 0, Skipped: 13

=== details ===
```
Running io.druid.query.lookup.LookupSnapshotTakerTest
Tests run: 5, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 0.018 sec <<< FAILURE! - in io.druid.query.lookup.LookupSnapshotTakerTest
testIOExceptionDuringLookupPersist(io.druid.query.lookup.LookupSnapshotTakerTest)  Time elapsed: 0.01 sec  <<< FAILURE!
java.lang.AssertionError: Expected exception: com.metamx.common.ISE
        at org.junit.internal.runners.statements.ExpectException.evaluate(ExpectException.java:32)
        at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
        at org.junit.rules.ExternalResource$1.evaluate(ExternalResource.java:48)
        at org.junit.rules.RunRules.evaluate(RunRules.java:20)
        at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
        at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
        at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:50)
        at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
        at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
        at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
        at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
        at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
        at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
        at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:283)
        at org.apache.maven.surefire.junit4.JUnit4Provider.executeWithRerun(JUnit4Provider.java:173)
        at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:153)
        at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:128)
        at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:203)
        at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:155)
        at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
```

--- a/processing/src/main/java/io/druid/query/lookup/LookupSnapshotTaker.java
+++ b/processing/src/main/java/io/druid/query/lookup/LookupSnapshotTaker.java
@@ -37,7 +37,7 @@ import java.util.List;
 public class LookupSnapshotTaker
 {
   private static final Logger LOGGER = new Logger(LookupSnapshotTaker.class);
-  private static final String PERSIST_FILE_NAME = "lookupSnapshot.json";
+  protected static final String PERSIST_FILE_NAME = "lookupSnapshot.json";
 
   private final ObjectMapper objectMapper;
   private final File persistDirectory;

--- a/processing/src/test/java/io/druid/query/lookup/LookupSnapshotTakerTest.java
+++ b/processing/src/test/java/io/druid/query/lookup/LookupSnapshotTakerTest.java
@@ -73,7 +73,10 @@ public class LookupSnapshotTakerTest
   public void testIOExceptionDuringLookupPersist() throws IOException
   {
     File directory = temporaryFolder.newFolder();
-    directory.setReadOnly();
+    File snapshotFile = new File(directory, LookupSnapshotTaker.PERSIST_FILE_NAME);
+    Assert.assertFalse(snapshotFile.exists());
+    Assert.assertTrue(snapshotFile.createNewFile());
+    Assert.assertTrue(snapshotFile.setReadOnly());
     LookupSnapshotTaker lookupSnapshotTaker = new LookupSnapshotTaker(mapper, directory.getAbsolutePath());
     LookupBean lookupBean = new LookupBean();
     lookupBean.name = "name";

